styles/formative/catppuccin.user.css

```diff
@@ -81,1054 +81,665 @@
-          background-color: @surface1 !important;
-          color: @text;
-          border-color: @overlay0;
-          .FormativeStudentHeader__InfoItem-sc-yw00ql-7 {
-            border-color: @overlay0;
-          }
-          .Button__StyledButton-sc-aum9f1-1 {
-            border-color: @accent-color;
-          }
+      .DragAndDropImageDropLocation__DropLocationContainerDiv-sc-oq3fcx-1 {
+        .DragAndDropImageDropLocationPoint-sc-1v0z50f-0 {
+          background: @surface0;
+          border-color: @surface0;
         }
-      }
-      /**********************************************************   */
-      #student-main-content {
-        background-color: @base;
-        .FormativeDescription__RootDiv-sc-msmcgd-0 {
-          * {
-            color: @text;
-          }
+        .dropzone-tip::before {
+          background: @surface0;
         }
-        .FormativeUnavailableMessage__InnerDiv-sc-14kxg7t-1 {
-          background-color: @mantle;
+        .dropzone {
           border-color: @accent-color;
-          i[name="warning"] {
-            color: @accent-color;
-          }
-          color: @text;
-        }
-        .FormativeStudentFooter__BottomDiv-sc-6tj89c-3 {
-          * {
-            color: @text;
+          background: @surface0;
+          .dropzone-number {
+            background: @surface1;
           }
         }
       }
-    }
-    .FormativeStudentHeader__NavStripDiv-sc-yw00ql-10 {
-      border-color: @overlay0;
-      background-color: @surface0 !important;
-      .FormativeProgressBarStyles__NavItemGroupDiv-sc-1rm72lt-2 {
+      .DragAndDropChoice__RootDiv-sc-1wzeg1h-3 {
+        background: @surface0;
         border-color: @overlay0;
-        background-color: @surface0;
-        &:hover {
-          background-color: @surface1;
-          .formative-nav-item {
-            color: @accent-color;
-            i {
-              color: @accent-color;
-            }
-            &:hover {
-              background-color: @surface2;
-            }
-          }
+      }
+      .AnswerCorrectnessIndicator__CircleDiv-sc-18hnk99-1 {
+        &.correct {
+          background: @green;
+        }
+        .material-icons-outlined {
+          color: var(--contrast-accent-color);
         }
       }
-      .formative-nav-item {
+      .ShortAnswerViewableDetails__InputAutosizeTextarea-sc-1ugk4gk-0 {
+        background: @surface0;
+        border-color: @overlay0;
         color: @text;
-        i {
+      }
+    }
+    .MediaSlider__RootSpan-sc-1gz67jn-0 {
+      .TrimmerMarker__TooltipDiv-sc-1f17hi4-1 {
+        background: @surface2;
+        color: @text;
+      }
+      .Thumb__RootSpan-sc-yv6w5k-0 {
+        background: @accent-color;
+        z-index: 2;
+      }
+      .Segment__SegmentSpan-sc-r0cy9o-0 {
+        background: @surface0 !important;
+      }
+      .TrimmerMarker__RootSpan-sc-1f17hi4-0 {
+        background: @overlay0;
+        z-index: 1;
+      }
+    }
```

Comment:
```suggestion
    .MediaSlider__RootSpan-sc-1gz67jn-0 {
      .TrimmerMarker__TooltipDiv-sc-1f17hi4-1 {
        background: @surface2;
        color: @text;
      }
      .Thumb__RootSpan-sc-yv6w5k-0 {
        background: @accent-color;
      }
      .Segment__SegmentSpan-sc-r0cy9o-0 {
        background: @surface0 !important;
      }
      .TrimmerMarker__RootSpan-sc-1f17hi4-0 {
        background: @overlay0;
      }
    }
```

Reply:
The `z-index`s are needed to fix this issue in the player. Is it not allowed to do so?
![image](https://github.com/catppuccin/userstyles/assets/45221816/5be1d67a-ddaf-4fdb-ba2b-a3783db9dba7)